gson/src/test/java/com/google/gson/functional/FormattingStyleTest.java

```diff
@@ -113,7 +113,16 @@ public void testFormatPretty() {
     String expectedJson = buildExpected("\n", "  ", true);
     assertThat(json).isEqualTo(expectedJson);
     // Sanity check to verify that `buildExpected` works correctly
-    assertThat(json).isEqualTo("{\n" + "  \"a\": [\n" + "    1,\n" + "    2\n" + "  ]\n" + "}");
+    assertThat(json)
+        .isEqualTo(
+            // spotless:off
```

Comment:
I don't think this should be necessary. If you put an empty `//` comment at the end of any line then google-java-format will not try to join them:
```java
assertThat(json).isEqualTo(
    "{\n" //
    + ...);
```
Though I don't really know whether the Spotless plugin applies exactly the same logic.

Reply:
> If you put an empty `//` comment at the end of any line then google-java-format will not try to join them

Yes that works, and I guess here it is ok because there are always multiple lines with `//`, but a single trailing `//` on its own looks to me rather like someone forgot to finish a comment.

> Though I don't really know whether the Spotless plugin applies exactly the same logic.

I think it should. If I understand it correctly it just delegates the formatting to google-java-format (and optionally applies additional formatting steps).